src/subscriber.ts

```diff
@@ -184,6 +211,25 @@ export class Message {
     }
   }
 
+  /**
+   * Acknowledges the message, expecting a response (for exactly once subscriptions).
+   *
+   * @example
+   * ```
+   * subscription.on('message', async (message) => {
```

Comment:
Can we document what happens when a non exactly once sub calls this method? I assume that the promise should always return success.